src/expr/src/scalar/func.rs

```diff
@@ -3217,6 +3233,7 @@ pub enum UnaryFunc {
     Sleep(Sleep),
     RescaleNumeric(u8),
     PgColumnSize(PgColumnSize),
+    PgGetConstraintdef(PgGetConstraintdef),
```

Comment:
This is very cool, didn't know that the new way let you share impls with different numbers of args so easily!